contracts/rewards/src/state.rs

```diff
@@ -42,9 +42,10 @@ impl EpochTally {
         }
     }
 
+    /// IMPORTANT: worker must be a valid address
```

Comment:
what does that mean?

Reply:
This just means the address should be validated before being passed in (usually via `deps.api.addr_validate(...)?`). It shouldn't be created via `Addr::unchecked()` or accepted blindly from user input.